airflow/sensors/external_task.py

```diff
@@ -341,6 +341,7 @@ def execute(self, context: Context) -> None:
                     states=self.allowed_states,
                     trigger_start_time=utcnow(),
                     poll_interval=self.poll_interval,
+                    timeout=self.timeout,
```

Comment:
@Xiroo  `self.timeout` is getting initialized from `conf.getfloat("sensors", "default_timeout"),` in base sensor and it's default value is `Sensor default timeout, 7 days by default (7 * 24 * 60 * 60).` I don't know what should be a meaningful default but this will be a drastic change for people using the existing default value of 60 sec kind of breaking change IMO.

cc: @uranusjr @hussein-awala

Reply:
I think 60 seconds don't have any meaning because it is the number fixed at 60 seconds not user's choice. Most of the users don't think too much about it and use it because it is just a fixed value. Rather, it makes users like me wonder why the deferrable mode's timeout is 60 seconds and not be able to find the answer in the docs, so it make users look at the code like I did.

IMHO, 60 seconds is not a proper value for default value since it does not raise any kind of time error(in general case) when using the sensor with default value. It can be confusing for users when the numbers aren't consistent.

Reply:
@utkarsharma2 Should I change to set default value to 60 sec? 
However, I don't understand the meaning of "people 'using' the existing default value of 60 sec kind of breaking change". It's just a default and fixed value so no one could choose to use it.
Also, they even don't know there's default value of 60 if they don't look at the source code or failure log. (Even if they saw the failure log or source code, they can't change it)

Reply:
@Xiroo I saw the other comments and as long as we add a new parameter to get the value from the user and they are aware of this change it should be fine. Also, like the @uranusjr said it should be optional and the default of this can be 60sec as before.